docs/Developers/Executing_Tests.rst

```diff
@@ -34,11 +34,7 @@ You can then execute our tests with
 
 ::
 
-    $ py.test
-
-.. note::
-    If ``py.test`` seems to give errors, try running ``python3 -m pytest``
-    instead.
```

Comment:
I'd like to keep this. I saw @thisch's [comment](https://github.com/coala/coala/issues/4189#issuecomment-299944884) but this is immensely helpful for Windows users where setting a path is painstakingly difficult.